docs/pages/yaml-reference.md

```diff
@@ -380,7 +380,7 @@ This documentation effectively reflects the Java object tree under [RawServiceSp
 
     * `health-check`
 
-      Health checks are additional validation that your task is healthy, in addition to just the fact that its process is still running. This is an extra convenience for sitations where a service can enter a zombie state from which it can never return. For example, it might query an HTTP endpoint to validate that an HTTP service is still responding.
+      Health checks are additional validation that your task is healthy, in addition to just the fact that its process is still running. This is an extra convenience for citations where a service can enter a zombie state from which it can never return. For example, it might query an HTTP endpoint to validate that an HTTP service is still responding.
```

Comment:
I think this was intended to be corrected to `situations` rather than `citations`

Reply:
Agree. Thank you